static/app/components/search/list.tsx

```diff
@@ -64,13 +62,17 @@ function List({
     [registerItemCount, resultList.length]
   );
 
+  // Show loading indicator if we're still loading and don't have any results yet
+  const hasResults = results.length > 0;
+  const isLoading = !allLoaded && !hasResults;
+
   return (
     <DropdownBox className={dropdownClassName}>
       {isLoading ? (
         <LoadingWrapper>
           <LoadingIndicator mini hideMessage relative />
         </LoadingWrapper>
-      ) : hasAnyResults ? (
+      ) : hasResults ? (
```

Comment:
nit: Should there be an indicator that more results are being fetched so that they don't pop in unexpectedly? I see it won't affect their ordering (resolvedTs was a smart call) but maybe a row item version of a loading indicator that says 'Searching [whatever]...' or something.

Reply:
I would take this PR as is, I toggle dark/light so often and this would be amazing -- feel free to punt on this if it overcomplicates^

Reply:
I'll add this after. API query searching is almost completely broken right now anyway so I am trying to see if I can't fix that first.